### DISCUSSION TITLE
Open an extra window

### DISCUSSION THREAD

**Discussion by LucasLopesCodemaker** [microsoft/react-native-windows#12905] [Q&A] [answered]
Hello, how are you? I'm developing an application for windows, with frameworReact Native for windows. My problem in this project is that I can't make an extra window appear when I click on a button. However, I "discovered" that if I use a native module I might be able to open that window. Unfortunately, I've been working on it for a week and I can't finish this part of the project. Is it really possible to do what I need? Or is it better to use Electron, for example? Thanks for your help!

**Comment by chrisglein** [comment, marked as answer]
We don't today have a sample that goes over creating multiple windows, but it's a totally reasonable request! (this came up recently in this issue: https://github.com/microsoft/react-native-windows/issues/12927). The short answer is that you use the native APIs to create a window and then put a React Native root view into that. Here's the issue to track creating a sample: https://github.com/microsoft/react-native-windows-samples/issues/757
